infrastructure/tests/priority-class.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: placeholder-priority
```

Comment:
Do we really need the priority class?

Reply:
Will this pre-empt the placeholder pods when the firsrt module is installed? If not I think we should remove it